src/content/en/showcase/2017/santa.md

```diff
@@ -0,0 +1,217 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2017-03-26T18:00:00.000Z #}
+{# wf_updated_on: 2017-03-26T18:00:00.000Z #}
+{# wf_featured_image: /web/showcase/2017/images/santa/featured.png #}
+{# wf_featured_snippet: Santa Tracker is a fun and educational experience for everyone—a holiday tradition here at Google. In 2016, we upgraded the Santa Tracker site to support offline via Service Worker. #}
+{# wf_tags: pwa,polymer,casestudy #}
+
+
+# Santa Tracker as a PWA {: .page-title }
+
+*By Sam Thorogood, Chief Santaware Engineer, Google—with contributions from Philipp Legner, Software Engineer, Google*
+
+<a class="button button-primary" href="https://santatracker.google.com/">
+  View the site
+</a>
+
+## Results
+
+* Santa is a PWA supporting Add To Home Screen (ATHS) and offline
+* 10% of eligible sessions began via the ATHS icon
+* 75% of users natively supported Custom Elements and Shadow DOM, two core parts of Web Components
+* [Lighthouse score of 81](https://pwa-directory.appspot.com/pwas/5704513654620160)
+* Offline through Service Worker is tied with lazy loading to only cache visited scenes and silently upgrading them on new releases
+
+## Background
+
+[Santa Tracker](https://santatracker.google.com) is a holiday tradition here at Google.
+Every year, you can celebrate the season with games and educational experiences all throughout December.
+And while Santa takes a well-deserved break, the elves work to release Santa Tracker as open-source—both on [the web](https://github.com/google/santa-tracker-web) and for [Android](https://github.com/google/santa-tracker-android).
+
+On the web, Santa Tracker is a large, interactive site with many unique 'scenes'—written using Polymer—that supports most modern browsers.
+This is determined through feature detection: Santa requires [`Set`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Set) and the [Web Performance API](https://developer.mozilla.org/en-US/docs/Web/API/Performance/now), among others.
```

Comment:
Done: "The assessment of whether a user's browser is 'modern' is determined through feature detection: Santa requires ..."